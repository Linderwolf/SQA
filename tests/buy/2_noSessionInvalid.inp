Hello1234
buy