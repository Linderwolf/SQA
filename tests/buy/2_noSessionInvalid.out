"Hello1234" is not a valid command. Please enter a valid Transaction code: login
"buy" is not a valid command. Please enter a valid Transaction code: login